disk.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import pyudev
+import psutil
+import subprocess
+
+# of course, we can use sys.argv but just for quick test this more useful
+freeSpaceGB = 10
+fileName = "testFile"
+fileSizeMB = 10
+
+context = pyudev.Context()
```

Comment:
1. `pyudev` depends on libudev, i.e. it will run on modern Linux distros only. Not on older Linuxes, FreeBSD or MacOS for example. 

2. How to find filesystems created on locally attached devices (not only removable one):
```
# mount |grep ^/dev
/dev/sda2 on / type ext3 (rw,relatime,data=ordered)
/dev/sda1 on /boot type ext3 (rw,relatime,data=ordered)
/dev/mapper/rsws14_vg-lv_images on /images type ext4 (rw,relatime,data=ordered)
/dev/mapper/rsws14_vg-lv_docker_xfs on /var/lib/docker-bk type xfs (rw,relatime,attr2,inode64,noquota)
```


3. Your program produces no output, no error messages on my Linux server:
```
admin@rsws14:tmp$ python disk.py
```
According to `strace python disk.py` output, your program tries to find removable disks, not able to find one and exits quietly:  `removable` list is empty.

4. In general, was enough to iterate over `psutil.disk_partitions()` list